src/main/java/edu/greenblitz/robotName/subsystems/limelight/MultiLimelight.java

```diff
@@ -38,9 +38,7 @@ public List<Optional<Pair<Pose2d, Double>>> getAll2DEstimates() {
 		ArrayList<Optional<Pair<Pose2d, Double>>> estimates = new ArrayList<>();
 		for (Limelight limelight : limelights) {
 			if (limelight.hasTarget()) {
-				if (limelight.getTagConfidence()) {
```

Comment:
Why?

Reply:
@aaa-png

Reply:
I delete it to test the pure vision data. We need to return it back to the code